src/core/include/openvino/op/lstm_sequence.hpp

```diff
@@ -17,103 +17,6 @@
 
 namespace ov {
 namespace op {
-namespace v0 {
-
-///
-/// \brief      Class for lstm sequence node.
-///
-/// \note       It follows notation and equations defined as in ONNX standard:
-///             https://github.com/onnx/onnx/blob/master/docs/Operators.md#LSTM
-///
-/// \sa         LSTMCell, RNNCell, GRUCell
-///
-///
-/// \ingroup ov_ops_cpp_api
-class OPENVINO_API OPENVINO_DEPRECATED(
-    "The class ov::op::v0::LSTMSequence is deprecated. It will be removed in 2025.0 release. Use "
```

Comment:
Please don't merge before: 2025.0
    "The class ov::op::v0::LSTMSequence is deprecated. It will be removed in 2025.0 release..."

Reply:
PR is 2025.0 milestone